Open Judge System/Web/OJS.Web/Views/Shared/_AdvancedSubmissionsGridPartial.cshtml

```diff
@@ -93,4 +93,24 @@
             counter++;
         });
     }
+</script>
+<script>
```

Comment:
Why open another script tag?

Reply:
To separate them logically.

Reply:
We do not have such a convention. The only thing that separates scripts is different files.